sys/unix/hints/macosx-nle

```diff
@@ -81,7 +81,7 @@ endif
 ifdef WANT_WIN_RL
 #CFLAGS += -URL_GRAPHICS -DRL_GRAPHICS -DNOUSER_SOUNDS
 CXXFLAGS += -std=c++14 -stdlib=libc++ -mmacosx-version-min=10.7
-LINK=g++
+LINK=clang
```

Comment:
I'd have assumed we'd need `clang++` here but if this works that's fine too. :)

Reply:
No, you are right, it's a fair point. I should stop abusing clang's cleverness.

Fixed.